cmd/ceremony/cert.go

```diff
@@ -87,6 +86,15 @@ const (
 	requestCert
 )
 
+// Subject returns a pkix.Name from the appropriate certProfile fields
+func (profile *certProfile) Subject() pkix.Name {
+	return pkix.Name{
+		CommonName:   profile.CommonName,
+		Organization: []string{profile.Organization},
+		Country:      []string{profile.Country},
+	}
+}
+
 func (profile *certProfile) verifyProfile(ct certType) error {
 	if ct != requestCert {
```

Comment:
I'd swap the sections of this if/else to be "if this is a crl: ..., otherwise: ...". Now that the "else" clause is much larger than the "if" clause, I think it makes sense to swap the order.

Reply:
Addressed in new PR #5326